src/safeds/ml/metrics/_classifier_metric.py

```diff
@@ -0,0 +1,10 @@
+from enum import Enum
+
+
+class ClassifierMetric(Enum):
```

Comment:
Consider implementing this as literal to be consistent with the current state of the library.